runtime/src/bank.rs

```diff
@@ -5909,6 +5905,13 @@ impl Bank {
         Some(vote_account.clone())
     }
 
+    /// Get the EpochStakes for the current Bank::epoch
+    pub fn current_epoch_stakes(&self) -> &EpochStakes {
+        self.epoch_stakes
+            .get(&self.epoch.saturating_add(1))
```

Comment:
Maybe add https://github.com/anza-xyz/agave/pull/2650#discussion_r1722533919 as a comment here for why this is doing `saturating_add(1)`.

Reply:
done